### PR TITLE
Sign Up modal

### LINKED ISSUE #22: Sign Up Modal
image

### PR DESCRIPTION
**Pull request by mustafaguldag**
Hi, I have built sign up modal. I did upload Yup package for Formik(Rahaf and ayşe are using it). #22 
image
image

### CODE REVIEW

**Review by ayse8888 — APPROVED** [review]
Hi Mustafa, the visual and the code seems nice !

File: src/components/NavBar/NavBar.js [hunk 1]
```diff
@@ -1,43 +1,52 @@
 import React from "react";
+import { useState } from "react";
 import { NavLink } from "react-router-dom";
 import { Navbar, Nav, Button, Form } from "react-bootstrap";
+import SignUpModal from "../SignUpModal/SignUpModal";
 import logo from "../../images/logo.png";
 import "./NavBar.css";
 
 function NavBar() {
+  const [modalShow, setModalShow] = useState(false);
```
Comment (halitbatur) [inline src/components/NavBar/NavBar.js]: maybe a better state name here ?

**Review by halitbatur — CHANGES_REQUESTED** [review]
Good Work Mustafa! this branch need updates and conflicts needs to be merged

File: src/components/NavBar/NavBar.js [hunk 2]
```diff
@@ -4,9 +4,11 @@ import { Navbar, Nav, Button, Form } from "react-bootstrap";
 import logo from "../../images/logo.png";
 import Login from "../Login/Login";
 import "./NavBar.css";
+import SignUpModal from "../SignUpModal/SignUpModal";
 
 function NavBar() {
   const [showLogin, setShowLogin] = useState(false);
+  const [openModal, setOpenModal] = useState(false);
```
Comment (halitbatur) [inline src/components/NavBar/NavBar.js]: still the state name needs to be better maybe smth that relates to signUpModal ?

**Review by halitbatur — CHANGES_REQUESTED** [review]

File: src/components/NavBar/NavBar.js [hunk 3]
```diff
@@ -4,9 +4,11 @@ import { Navbar, Nav, Button, Form } from "react-bootstrap";
 import logo from "../../images/logo.png";
 import Login from "../Login/Login";
 import "./NavBar.css";
+import SignUpModal from "../SignUpModal/SignUpModal";
 
 function NavBar() {
   const [showLogin, setShowLogin] = useState(false);
+  const [showSignUp, setShowSignUp] = useState(false);
```
Comment (halitbatur) [inline src/components/NavBar/NavBar.js:11]: we need one extra line after this

**Review by halitbatur — APPROVED** [review]
one more line and then its ready to be merged

File: src/components/NavBar/NavBar.js (hunk 3 above)
Comment (mustafaguldag) [reply src/components/NavBar/NavBar.js:11]: Thanks! I fixed that.

**Pull request merged**